graph2tac/tfgnn/predict.py

```diff
@@ -157,11 +158,26 @@ def __init__(self,
         self._allocation_reserve = allocation_reserve
 
         # create dummy dataset for pre-processing purposes
-        graph_constants_filepath = log_dir / 'config' / 'graph_constants.yaml'
-        with graph_constants_filepath.open('r') as yml_file:
-            graph_constants_d = yaml.load(yml_file, Loader=yaml.UnsafeLoader)
-            graph_constants_d['data_config'] = DataConfig(**graph_constants_d['data_config'])
-            graph_constants = GraphConstants(**graph_constants_d)
+        graph_constants_filepath_yaml = log_dir / 'config' / 'graph_constants.yaml'
+        graph_constants_filepath_json = log_dir / 'config' / 'graph_constants.json'
+        if not graph_constants_filepath_json.exists():
+            logger.info(f'no json graph_constants file, trying to load yaml')
+            with graph_constants_filepath_yaml.open('r') as yml_file:
+                graph_constants_d = yaml.load(yml_file, Loader=yaml.UnsafeLoader)
```

Comment:
Can you make this SafeLoader?  All our yaml should be safe now and this prevents code execution.

Reply:
ok, I just copied the original yaml-loading code, I will change it to SafeLoader